crates/language/src/language_registry.rs

```diff
@@ -1065,6 +1081,26 @@ impl LanguageRegistryState {
     }
 }
 
+#[derive(Clone, Default)]
+struct DapBinaryStatusSender {
+    txs: Arc<Mutex<Vec<mpsc::UnboundedSender<(LanguageServerName, LanguageServerBinaryStatus)>>>>,
```

Comment:
Could we use [`crossbeam::channel`](https://docs.rs/crossbeam/latest/crossbeam/channel/index.html)? It's MPMC too.

Reply:
We can't do that, mea culpa